config/crd/bases/packages.glasskube.dev_packageinfos.yaml

```diff
@@ -191,6 +191,26 @@ spec:
                       chartVersion:
                         description: ChartVersion of the chart that should be installed.
                         type: string
+                      releases:
+                        items:
+                          properties:
+                            chartName:
```

Comment:
How is it possible to define different namespaces for these charts, as recommended by the istio installation?

Reply:
It's not possible. If you are talking about the istio ingress, it is an optional component and I would create a separate package for it.

Reply:
perfect